src/pages/Statistics/StatisticsCities.tsx

```diff
@@ -230,8 +231,16 @@ const StatisticsCities = () => {
 
     // seting (for chart needs) statisticsItems indicators of the very first element
     // because they are the same for all the elements
+    let entryToSetIndicators = response.data.find((entry: any) => entry.yearStatistics.length != 0)
```

Comment:
can we use some type to `entry`?

Reply:
clarified corresponding types in new commits